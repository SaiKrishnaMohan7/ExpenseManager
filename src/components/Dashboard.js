import React from 'react';

const Dashboard = () => {
    return (
    <div>
        this i from the dashboard component
    </div>        

    );
};

export default Dashboard;