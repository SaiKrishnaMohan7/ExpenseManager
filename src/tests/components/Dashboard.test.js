import React from 'react';
import { shallow } from 'enzyme';

import Dashboard from '../../components/Dashboard';

test('should render Dashboard component with ExpenseListFilters and ExpenseList', () => {
  const wrapper = shallow(<Dashboard />);

  expect(wrapper).toMatchSnapshot();
});