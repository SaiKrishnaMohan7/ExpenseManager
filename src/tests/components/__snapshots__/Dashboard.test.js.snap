// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard component with ExpenseListFilters and ExpenseList 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
